### ISSUE TITLE
3 is a charm

### ISSUE THREAD

**Issue by TomRausner** [ubports/ubports-installer#1898]
**UBports Installer `0.8.7-beta` (snap ?)**
Environment: `Ubuntu Groovy Gorilla`
Device: frieza
Target OS: Ubuntu Touch
Settings: `undefined`
OPEN-CUTS run: https://ubports.open-cuts.org/run/606ca2b17327390007b9d575
Log: *N/A*

Well, the installation failed to catch my device twice, but worked like a charm at last.

**Comment by UniversalSuperBox** [comment]
Hi @TomRausner,

Did you have trouble with device being detected by the installer or something else?

**Comment by TomRausner** [comment]
Yes... sort of. I plugged the tablet in twice without the installer
discovering it but, as I wrote; third time it worked smoothly -no snags of
any kind.

Tom
----------------------------------------------
Frihed Kærlighed Fællesskab
----------------------------------------------
